index.html

```diff
@@ -667,6 +667,50 @@ <h4>Use the new JSON-LD context when issuing new credentials</h4>
 
       </section>
 
+      <section>
+        <h3>Extending JWTs</h3>
+
+        <p>
+The VC Data Model specifies a minimal set of JWT <a>claim</a> names that must be used
+to represent <a>verifiable credential</a> properties and <code>credentialSubject</code>
+properties. Implementors may wish to extend a <a>verifiable credential<a> with a
+property that is either new e.g. drivingLicenseNumber, or that is already defined 
+as a JWT claim with IANA e.g.<a>given_name</a>. It is recommended that these properties
+are placed directly in either the JWT <code>vc</code> <a>claim</a> or
+<code>credentialSubject</code> property of the <code>vc</code> <a>claim<a> as appropriate.
```

Comment:
This is just about the paragraph above, based on [the latest Data Model revision](https://github.com/w3c/vc-data-model/pull/711#discussion_r313964622). Further grammar, punctuation, markup, and other changes *will* be needed to the remainder of this PR (#41), whether or not that Data Model revision is adopted, but it's not worth making all those changes here until that PR is concluded.

```
The VC Data Model specifies a minimal set of JWT <a>claim</a> names that must be used
to represent <a>verifiable credential</a> properties and <code>credentialSubject</code>
properties. Implementors may wish to extend a <a>verifiable credential<a> with a
property that is either new e.g. drivingLicenseNumber, or that is already defined 
as a JWT claim with IANA e.g.<a>given_name</a>. It is recommended that these properties
are placed directly in either the JWT <code>vc</code> <a>claim</a> or
<code>credentialSubject</code> property of the <code>vc</code> <a>claim<a> as appropriate.
```
->
```
The VC Data Model specifies a minimal set of JWT <a>claim</a> names which are
to be used to represent the <a>properties</a> of a <a>verifiable credential</a>
and its <code>credentialSubject</code>. Implementors may wish to extend a
<a>verifiable credential<a> with some <a>properties</a> that are new (e.g.,
<code>drivingLicenseNumber</code>, <code>mySpecialProperty</code>) or that are
already registered with IANA as JWT claims (e.g., <code>given_name</code>,
<code>phone_number_verified</a>).
        </p>
        <p>
As the Data Model Specification states, such extension properties MUST be
placed either directly in the JWT <code>vc</code> <a>claim</a> or in the
<code>credentialSubject</code> property of the <code>vc</code> <a>claim<a> as
appropriate. <!-- Should link to the relevant section of the data model spec. -->
```

Reply:
@TallTed Your comment has been superceded by the revised text added to the data model. Can you review this PR again please based on the latest version of the data model.

Reply:
This, with the appropriately formatted link to the "JWT Encoding" subsection of "JWT and JWS Considerations", should still replace the paragraph above.  I've replaced the one `MUST` with `are to be` to remove any concerns about normative language.

```
The VC Data Model specifies a minimal set of JWT <a>claim</a> names which are
to be used to represent the <a>properties</a> of a <a>verifiable credential</a>
and its <code>credentialSubject</code>. Implementers may wish to extend a
<a>verifiable credential<a> with some <a>properties</a> that are new (e.g.,
<code>drivingLicenseNumber</code>, <code>mySpecialProperty</code>) or that are
already registered with IANA as JWT claims (e.g., <code>given_name</code>,
<code>phone_number_verified</a>).
        </p>
        <p>
As the Data Model Specification states, such extension properties are to be
placed either directly in the JWT <code>vc</code> <a>claim</a> or in the
<code>credentialSubject</code> property of the <code>vc</code> <a>claim<a> as
appropriate. <!-- Should link to the relevant section of the data model spec. -->
```